docs/dev/README.md

```diff
@@ -1,36 +1,101 @@
+## *IntelliJ*
 
-### Gradle - Building and Running the Code
+Plugins:
+  - *Eclipse Code Formatter* 
+    - configure in settings to pick up the eclipse xml formatter and import ordering files.
+  - *checkstyle-IDEA*
+  - *Save Actions*
+    - configure in settings to add 'final' to local variables and class variables.
 
-Useful build commands:
+# Team Communication
 
-```
-./gradlew test
-./gradlew run
-```
+*Github Issues*: https://github.com/triplea-game/triplea/issues
+- Primary means of communication, conversations are open and recorded for historical reference.
+- Open, allows for team participation and task balancing
+
+*Gitter*: https://gitter.im/triplea-game/social
+
+Quick 1:1 messages, administrative informational broadcast messages. 
+For example: "Restarting bot 15, looks to be on fire"
+  
+
+*Forum*: http://forums.triplea-game.org
+
+Communication to the larger TripleA community. Feature and game changes discussions would belong here.
+
+
+# Code Review 
+
+### PR Guidelines
+- describe functional changes
+- describe what was tested
+- describe refactoring that was done
+- if UI changes were made, include screen shots giving an overview of how things look
+- code contributors should test their updates quite thoroughly
+
+### Code reviewer expectation:
+- shall not "self-review" and "self-merge" their own work 
+
+### Requirements for someone to have write access
+- contribute some cod
+- provide valuable code reviews
+- sustained engagement over at least a few months
+
+### PR Merge Guidelines
+
+At least one code reviewer must:
+- try out any functional changes
+- do a thorough code check
+Before merge:
+- all comments questions should be answered/addressed
+
+### Code Review Ordering
+- Reviewers should start with the oldest PRs and work their way forward in time. Favor merging things in that order as well. The reason for the merge order is to make merge conflicts a bit more predictable, if you are the first PR in the queue, then there should be no merge conflicts. If 2nd, then in theory you would only have to worry about the one open PR request to cause merge conflicts.
+
+
+# Versioning 
+
+Simplified 3 number versioning system based on semantic versioning:
+
+ 
+``` release.compatability.build_number```
+
+
+**Release** - A version number used by the TripleA team for version partitioning. 
+**compatability** - Incremented whenever backwards compatibility with previous version is lost
+**Build** - auto-generated by the build system.
+
+
+* Last discussed in: https://github.com/triplea-game/triplea/issues/1739
+
+# Troubleshooting
 
-## Set up IDE Formatter
-See [code format](https://github.com/triplea-game/triplea/blob/master/docs/dev/code_format.md)
+Response to a frozen game problem, below are steps notes of what would be needed to debug such a problem:
 
-## Useful Tools
+  - launch the game from a console window, running: ./triplea_unix.sh  | tee output.txt
```

Comment:
I don't believe the _triplea_unix.sh_ script exists anymore.

Reply:
yeah, these notes are linux specific, not the best : |
Largely this is moved documentation, but it makes sense to go ahead and spend some time to prune, I removed the script mention in: d00b938